src/compat/eme/custom_media_keys/webkit_media_keys.ts

```diff
@@ -123,9 +123,14 @@ class WebkitMediaKeySession
           return reject("Unavailable WebKit key session.");
         }
         try {
+          const uInt8Arraylicense = license instanceof ArrayBuffer
+            ? new Uint8Array(license)
+            : license instanceof Uint8Array
+              ? license
+              : new Uint8Array((license as ArrayBufferView).buffer);
```

Comment:
Weird indent.

I guess that's OK, but I generally find it more readable that way:
```ts
const uInt8Arraylicense = license instanceof ArrayBuffer ? new Uint8Array(licence)) : 
                          license instance of Uint8Array ? license : 
                                                           new Uint8Array((license as ArrayBufferView).buffer);
```

Also, casting as ArrayBufferView should not be needed. Why is that the case? If it is, it means that we forgot to handle other cases (as TypeScript should be smart enough to notice that this cannot be an ArrayBuffer)

Reply:
Ho yes, thanks for the indent suggestion, since I'm using prettier, I was more used to write it like this, but it's totally fair that since we don't use prettier in this project we should use another indent style.

And yes, I forgot to edit the `update` method signature by this:
```
this.update = (license: Int8Array | ArrayBuffer) => {...}
```

So Typescript was not understanding cause we already checked that the license was a `UInt8Array`